internal/cache/cache.go

```diff
@@ -158,7 +158,12 @@ func (c *LoadingCache) getValue(key string) (interface{}, bool) {
 func (c *LoadingCache) Purge() {
 	c.mu.Lock()
 	defer c.mu.Unlock()
-	for k, v := range c.data {
+
+	// to release the memory, as otherwise old map would store same amount of entries to prevent reallocations
+	oldData := c.data
+	c.data = make(map[string]*cacheItem)
+
+	for k, v := range oldData {
 		delete(c.data, k)
```

Comment:
`c.data` is empty after line 164. There is no point to delete data from `oldData` either.